drivers/ovsdriver.go

```diff
@@ -377,44 +373,64 @@ func (d *OvsDriver) DeleteEndpoint(id string) (err error) {
 	return nil
 }
 
-// CreatePeerHost creates peer host object and adds VTEPs if necessary
-func (d *OvsDriver) CreatePeerHost(id string) error {
-	log.Infof("CreatePeerHost for %s", id)
+// AddPeerHost adds VTEPs if necessary
+func (d *OvsDriver) AddPeerHost(node core.ServiceInfo) error {
+	// Nothing to do if this is our own IP
+	if node.HostAddr == d.localIP {
+		return nil
+	}
+
+	log.Infof("CreatePeerHost for %+v", node)
```

Comment:
That’s fine, but please put in an issue for it.

-Erik

On 17 Sep 2015, at 19:52, Sukhesh Halemane wrote:

>> @@ -377,44 +373,64 @@ func (d *OvsDriver) DeleteEndpoint(id string) 
>> (err error) {
>> 	return nil
>> }
>>
>> -// CreatePeerHost creates peer host object and adds VTEPs if 
>> necessary
>> -func (d *OvsDriver) CreatePeerHost(id string) error {
>> -	log.Infof("CreatePeerHost for %s", id)
>> +// AddPeerHost adds VTEPs if necessary
>> +func (d *OvsDriver) AddPeerHost(node core.ServiceInfo) error {
>> +	// Nothing to do if this is our own IP
>> +	if node.HostAddr == d.localIP {
>> +		return nil
>> +	}
>> +
>> +	log.Infof("CreatePeerHost for %+v", node)
>
> These are very useful for debugging right now. Can we make them debug 
> messages later when things are stable.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/netplugin/pull/148/files#r39820440